src/charts/donut.js

```diff
@@ -556,6 +558,22 @@ define(function(require) {
 
         // API
 
+        /**
+         * Gets or Sets the centeredTextFunction of the chart. If function is provided
+         * the format will be changed by the custom function's value format.
+         * @param  {Function} _x        Custom function that returns a formatted string
+         * @return {Function | module}  Current centeredTextFunction or Chart module to chain calls
+         * @public
```

Comment:
I would add here some more info about how to use this, like the default format and the data that the user could expect to have within the 'd' object.